GOVERNANCE.md

```diff
@@ -52,6 +52,22 @@ cannot veto or remove items.
 The moderator is responsible for summarizing the discussion of each
 agenda item and sends it as a pull request after the meeting.
 
+## Attending External Standards Meetings
+
+At various times members of this committee will attend Standards Meetings
+at external organizations as a representative of the OpenJS Foundation.
+If a member would like to attend a meeting as a delegate of the OpenJS Foundation
+they should open an issue stating:
+
+* The standards meeting they wish to attend
+* The OpenJS Foundations relationship to this standards organization
+* The date and location of the meeting
+* If they will be attending in person or remotely
+* The estimated cost of their particiaption
```

Comment:
I don't think the estimated cost belongs here; should it not be a part of the CPC travel fund request instead, if the delegate is looking for financial support? Otherwise the implication is that the Standards group is going to approve travel funds, which probably is not the intent here.